### DISCUSSION TITLE
New user error

### DISCUSSION THREAD

**Discussion by dlvommen76** [mckaywrigley/chatbot-ui#1504] [Help]
Every time I start chatbot-ui I get this error. Does anyone know what I can do to fix this?
![Error](https://github.com/mckaywrigley/chatbot-ui/assets/81318079/98e6c6f3-f5a3-4415-b201-ad0449116de3)

**Comment by adityakulkarni0** [comment]
me too @mckaywrigley could you please help out with this? thank you

**Comment by mhalavanja** [comment]
You can check locally in your supabase if there is a `user_id` under your `profiles` table that is matching the `id` from `auth.users` table. It probably doesn't since you are getting this error. If that is the only problem, you can just update your profile with that id.

**Reply by JaswaniSaniya** [reply]
How do we check this ?

**Reply by mhalavanja** [reply]
Here you should see the tables http://localhost:54323/project/default/editor and check these two tables I mentioned. Also, try just restarting docker and supabase, that might help as well.

**Comment by joeri23** [comment]
I have the same issue and i've checked the user_id and they match :(

**Reply by mhalavanja** [reply]
If you have multiple profiles with the same `user_id` it will also error. If it is not that, I am really not sure just by looking at the code why would it error. I would look into DB config at that point, restart Docker and Supabase.

**Reply by joeri23** [reply]
after some time, it started working. I guess some patience was required as well ;)

**Reply by joeri23** [reply]
today i was messing around on my server and the error returned. I fixed it by using the servers ip in the api url instead of 127.0.0.1 in the .env.local
